crypto/Makefile

```diff
@@ -9,24 +9,21 @@ ci: fv
 .PHONY: fv
 fv: setup-fv bin/fips-test-build
 	# Run a server using the FIPS strict mode settings and wait 1s to let it come up.
-	./bin/fips-test-build &
+	$(DOCKER_RUN) --name fips-test-build --rm $(CALICO_BUILD) ./bin/fips-test-build &
```

Comment:
Can we use `-d` to run as a [background](https://docs.docker.com/engine/reference/run/#foreground-and-background) process?